app/controllers/subjects_controller.rb

```diff
@@ -25,11 +25,10 @@ def create
     @subject = Subject.new(subject_params)
     return unless authorize @subject
 
-    if @subject.save
-      render_success_response('success.', true)
-    else
-      render_failed_response('Failed to create subject.')
-    end
+    @subject.save!
```

Comment:
@josephelias-rubyians  Even when you do @subject.save without bang, it will have the errors in the object.

if @subject.save

end
@subject.errors will have the model validation errors.. 

Ideally you shouldn't do bang ! unless you absolutely need it.